tests/test_widgets.py

```diff
@@ -14,3 +18,15 @@ def test_is_hidden(self):
         from django_select2.widgets import HeavySelect2Widget
         new_widget = HeavySelect2Widget(data_url="/")
         assert new_widget.is_hidden is False
+
+
+class TestSelect2Widget(object):
+    url = reverse('select2_widget')
+
+    def test_selecting(self, db, client, live_server, driver):
+        driver.get(live_server + self.url)
+        with pytest.raises(NoSuchElementException):
+            error = driver.find_element_by_xpath('//body[@JSError]')
+            pytest.fail(error.get_attribute('JSError'))
+        elem = driver.find_element_by_id('s2id_id_number')
+        elem.click()
```

Comment:
what does the click do? Is there any assertion that can t be done to confirm the change is working?